src/app/location-map/map-modal/map-modal.component.html

```diff
@@ -1,8 +1,10 @@
 <div class="modal-header">
-  <button type="button" class="close pull-right" aria-label="Close" (click)="closeModal()">
-    <span aria-hidden="true">&times;</span>
-  </button>
+    <button type="button" class="close pull-right" aria-label="Close" (click)="closeModal()">
+        <span aria-hidden="true">&times;</span>
+    </button>
 </div>
 <div class="modal-body">
-    <app-location-map [coordsSet]="projectCoords" (getMap)="getMap($event)"></app-location-map>
+    <app-location-map *ngIf="modalShown"
+                      [lat]="lat" [lng]="lng">
+    </app-location-map>
```

Comment:
In this case, we don't need to edit the coordinates, so we don't use two-way binding. Notice the difference.